### DISCUSSION TITLE
Fetch release notes from github enterprise

### DISCUSSION THREAD

**Discussion by expe-elenigen** [renovatebot/renovate#23512] [Suggest an Idea]
### Tell us more.

Hi, from what I understand, we can only fetch the release notes from github.com, but in our case we have our own github enterprise solution and when we are releasing new versions for our libraries, renovate won't fetch the release notes.
There was a question a question about it, but it looks like it was never converted into a feature request:
https://github.com/renovatebot/renovate/discussions/15265

**Comment by rarkins** [comment]
Release notes on GHE are supported in general
